package/gluon-mesh-vpn-core/luasrc/usr/lib/lua/gluon/mesh-vpn.lua

```diff
@@ -44,4 +44,19 @@ function M.get_active_provider()
 	return nil, nil
 end
 
+function M.get_enabled_public_key()
+	if M.enabled() ~= true or uci:get_bool('gluon', 'mesh_vpn', 'pubkey_privacy') ~= true then
```

Comment:
This code is incorrect. `pubkey_privacy` is a site field, it never exists in UCI.